draft-ietf-ppm-dap.md

```diff
@@ -277,42 +282,42 @@ also follows {{RFC8446}}.
 
 # Overview {#overview}
 
-The protocol is executed by a large set of clients and a small set of servers.
-Servers are referred to as *aggregators*. Each client's input to the protocol is
-a set of measurements (e.g., counts of some user behavior). Given the input set
-of measurements `x_1, ..., x_n` held by `n` users, the goal of a protocol for
-privacy preserving measurement is to compute `y = F(p, x_1, ..., x_n)` for some
-function `F` while revealing nothing else about the measurements.
+The protocol is executed by a large set of Clients and a small set of servers.
+Servers are referred to as "Aggregators". Each Client's input to the protocol is
+its measurement (or set of measurements, e.g., counts of some user behavior).
+Given the input set of measurements `x_1, ..., x_n` held by `n` users, the goal
+of DAP is to compute `y = F(p, x_1, ..., x_n)` for some function `F` while
+revealing nothing else about the measurements. We call `F` the "aggregation
+function".
 
 This protocol is extensible and allows for the addition of new cryptographic
 schemes that implement the VDAF interface specified in
 {{!VDAF=I-D.draft-irtf-cfrg-vdaf-03}}. Candidates include:
 
-* Prio3, which allows for aggregate statistics such as sum, mean, histograms,
-  etc. This class of VDAFs is based on Prio {{CGB17}} and includes improvements
-  described in {{BBCGGI19}}.
+* Prio3 ({{Section 7 of !VDAF}}), which allows for aggregate statistics such as
+  sum, mean, histograms, etc.
 
-* Poplar1, which allows for finding the most popular strings among a collection
-  of clients (e.g., the URL of their home page) as well as counting the number
-  of clients that hold a given string. This VDAF is the basis of the Poplar
-  protocol of {{BBCGGI21}}, which is designed to solve the heavy hitters problem
-  in a privacy preserving manner.
+* Poplar1 ({{Section 8 of !VDAF}}), which allows for finding the most popular
+  strings among a collection of Clients (e.g., the URL of their home page) as
```

Comment:
"collection of Clients" is awkward here. It's a collection of their inputs, right?

Reply:
Yes, I was aiming for minimal change, but this seems like a sensible change. Changed to "set of".